flutter/shell/platform/tizen/BUILD.gn

```diff
@@ -183,7 +184,10 @@ template("embedder") {
         "dali2-core",
       ]
 
-      defines += [ "NUI_SUPPORT" ]
+      defines += [
+        "NUI_SUPPORT",
+        "CLIPBOARD_SUPPORT",
```

Comment:
Do we need `CLIPBOARD_SUPPORT`? Is there any Tizen version or profile that doesn't support the feature?

Reply:
It is not supported in Tizen 5.5 and 6.0 versions and wearable. In fact, since the conditions are the same as NUI_SUPPORT, it can be reused with an appropriate name. But I think it's okay to keep it because it's functionally separate.